src/components/AuthContainer.js

```diff
@@ -13,17 +13,17 @@ import TermsOfService from 'src/pages/TermsOfService'
 
 const AuthContainer = ({ children }) => {
   const { name, public: isPublic } = useRoute()
-  const { isSignedIn, registrationStatus, acceptedTos, profile } = useStore(authStore)
+  const { isSignedIn, registrationStatus, acceptedTos, acceptedSamTos, profile } = useStore(authStore)
   const authspinner = () => h(centeredSpinner, { style: { position: 'fixed' } })
 
   return Utils.cond(
     [isSignedIn === undefined && !isPublic, authspinner],
     [isSignedIn === false && !isPublic, () => h(SignIn)],
     [registrationStatus === undefined && !isPublic, authspinner],
     [registrationStatus === 'unregistered', () => h(Register)],
+    [(acceptedTos === undefined || acceptedSamTos === undefined) && !isPublic, authspinner],
```

Comment:
Some comments as to what these cases represent would be helpful. As someone who is new to this code, I don't know why `isPublic` matters, and what the name `privacy` signifies.